### PR TITLE
Fixed minor debiased lasso and sparse DML bugs

### PR DESCRIPTION
**Pull request by moprescu**
Removed `positive` from the list of valid attributes for the debiased lasso and multioutput debiased lasso. Made common base class LinearModelFinalCateEstimatorMixin.

### CODE REVIEW

File: econml/cate_estimator.py [hunk 1]
```diff
@@ -381,12 +381,12 @@ def effect(self, X=None, *, T0=0, T1=1):
     effect.__doc__ = BaseCateEstimator.effect.__doc__
 
 
-class StatsModelsCateEstimatorMixin:
+class LinearModelFinalCateEstimatorMixin(BaseCateEstimator):
+    """Base class for models where the final stage is a linear model."""
 
     def _get_inference_options(self):
```
Comment (vasilismsr) [inline econml/cate_estimator.py]: I think you can just remove this implementation here.

Comment (moprescu) [reply econml/cate_estimator.py]: no... we talked about this. Both `StatsModelsCateEstimatorMixin` and `DebiasedLassoCateEstimatorMixin` now inherit from this common base class.

Comment (vasilismsr) [reply econml/cate_estimator.py]: I mean the implementation of _get_inference_options. It is already implemented in the BaseCateEstimator and you just repeat it here.

Comment (moprescu) [reply econml/cate_estimator.py]: oh, gotcha. It shall be removed.

**Pull request merged**